backend/app/usecase/match/contains_any.go

```diff
@@ -0,0 +1,25 @@
+package match
+
+import "strings"
+
+var _ Keyword = (*ContainsAny)(nil)
+
+// ContainsAny checks if an input contains any element of a list.
+type ContainsAny struct {
+}
+
+// IsMatch checks if the input contains any word.
+func (c ContainsAny) IsMatch(words []string, input string) bool {
+	if input == "" {
+		return false
+	} else if len(words) == 0 {
+		return false
```

Comment:
This here is correct, the `input` does not have any patterns given in the `words`. Again, this is also handled by the main logic below.